cpp/ps0001/p001-soln-cpp.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream> 
+using namespace std;
+int main() 
+{
+  // prompting user for 3 numbers to add 1/2
+  short number_one;
+  cin>>number_one;
+  short number_two;
+  cin>>number_two;
+  short number_three;
+  cin>>number_three;
+
+  // outputting the result of the summation process 3/3
+  short Summation=number_one+number_two+number_three;
+  cout<<"The sumation of three numbers ="<<Summation;
+  
+  return 0; 
+}
+// great work 😎 8/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code